test/common/cloudevents_vegeta_targeter.go

```diff
@@ -0,0 +1,104 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package common
+
+import (
+	cloudevents "github.com/cloudevents/sdk-go"
+	cehttp "github.com/cloudevents/sdk-go/pkg/cloudevents/transport/http"
+	vegeta "github.com/tsenart/vegeta/lib"
+	"math/rand"
+	"strconv"
+	"time"
+)
+
+func init() {
+	rand.Seed(time.Now().UnixNano())
+}
+
+var letterRunes = []rune("abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ")
+
+type CloudEventsTargeter struct {
+	sinkUrl          string
```

Comment:
Depending on the settings, the Go linter will complain about variables that have non capitalized "Url" in their name. Feel free to ignore this comment if code analysis doesn't throw any warning :)

Reply:
No blame from my linter :smile: